tests/unit/assessment/__init__.py

```diff
@@ -28,4 +29,5 @@ def workspace_client_mock(clusters="no-spark-conf.json"):
     ws = create_autospec(WorkspaceClient)
     ws.clusters.list.return_value = _load_list(ClusterDetails, f"../assessment/clusters/{clusters}")
     ws.cluster_policies.get = _cluster_policy
+    ws.pipelines.get.return_value = _load_list(GetPipelineResponse, "clusters/pipeline_cluster.json")[0]
```

Comment:
@prajin-29 , do you read the surrounding code before changing it? 🤦

why `ws.pipelines.get` is different than `ws.cluster_policies.get`? it's illogical!

Reply:
updated `ws.peipeline.get`  similar to `ws.cluster_policies.get` . With this I have included the pipeline spec configuration also inside the json so that we can access everything in one shot.